CHANGELOG.md

```diff
@@ -18,7 +18,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 -   Added Cancel button on edit screens
 -   Added tests for /registry/admin/registry/monitoringlocation/add/ and change
 -   Added message for bulk upload - Note: A user can only upload a CSV file
-
+-   Added Validation for Decimal values in Bulk Upload
```

Comment:
This is in the wrong place. It should be under the Unreleased tag.

Reply:
I will fix this and recommit

Reply:
oops.